code/modules/lighting/lighting_overlay.dm

```diff
@@ -6,6 +6,7 @@
 	anchored = 1
 	icon = LIGHTING_ICON
 	layer = LIGHTING_LAYER
+	plane = LIGHTING_PLANE
```

Comment:
Shameless repost

YES MAN AHHAAAHAHAHAHA PLANESSSSSS YEEESSSSSS!

THIS MIGHT MEAN https://github.com/ParadiseSS13/Paradise/pull/8668 HAS A CHANCE
@Aurorablade REJOICE